### ISSUE TITLE
Can this model run under tensorflow 1.13 and torch 1.0.1? 

### ISSUE THREAD

**Issue by lk1983823** [ermongroup/ssdkl#3]
Can this model run under tensorflow 1.13 and torch 1.0.1?

**Comment by nealjean** [comment]
I'm not sure, it has only been tested on tf 0.12.1 and pytorch 0.4.1

@sangmichaelxie do you know the answer?

**Comment by lk1983823** [comment]
Okay, I will try. Btw, can this model be accelerated using GPU

**Comment by sangmichaelxie** [comment]
The code doesn't work for newer versions of TF right now. It does use GPU.

**Comment by WeihongLi-ac** [comment]
> The code doesn't work for newer versions of TF right now. It does use GPU.
> […](#)

Hi, many thanks for providing code for all methods on semi supervised regression task. I've run the code and have several questions. 1. Even I enable the use_gpu flag, it seems that there are no any GPUs used. 2. The results look different from those shown in the paper (e.g., 2.04 for ssdkl using 100 labeled samples). 3. Is 'semisup' the ssdkl method? 4. Do you have the pytorch version for this method?

**Comment by sangmichaelxie** [comment]
You should check your LD_LIBRARY_PATH and associated environment variables
that cuda is correctly linked, and that tensorflow-gpu is installed
(without tensorflow also installed). For some of the methods (like label
prop) we didn't provide a GPU implementation. When we ran the experiments,
the results were stable - make sure you're using the provided config.yaml
and print_results.py script. Also notice that we report median "Relative to
DKL" results in the paper. Yes, semisup is the SSDKL method. We don't have
pytorch code.

**Comment by xueqings** [comment]
> > The code doesn't work for newer versions of TF right now. It does use GPU.
> > […](#)
> 
> Hi, many thanks for providing code for all methods on semi supervised regression task. I've run the code and have several questions. 1. Even I enable the use_gpu flag, it seems that there are no any GPUs used. 2. The results look different from those shown in the paper (e.g., 2.04 for ssdkl using 100 labeled samples). 3. Is 'semisup' the ssdkl method? 4. Do you have the pytorch version for this method?

Hi, do you have the pytorch version for this method right now? Thanks a lot.
